README.md

```diff
@@ -376,6 +376,36 @@ To disable the splashscreen add the following preference to `config.xml`:
 <preference name="SplashScreenDelay" value="0"/>
 ```
 
+##### Display app version (Android, iOS)
+
+To display your app version on splashscreen add the following preference to `config.xml`:
+```xml
+<preference name="ShowSplashScreenAppVersion" value="true" />
+```
+
+Now just configure your app version into the widget version attribute, like that : 
+```xml
+<widget version="1.0" >...</widget>
+```
+
+
+###### Prefix app version
+To set a prefix version add the following preference to `config.xml`:
+```xml
+<preference name="SplashScreenAppVersionPrefix" value="v" />
+```
+In this case you'll have `v1.0`
+
+
+###### Customize app version
```

Comment:
What font will be used? Maybe add an example screenshot here so one can visualize what it will look like.

Reply:
For the screenshot, could i commit picture into the repo ?
Or i need to upload file on the web ?

Reply:
Just can just paste images into comments - they will automatically be uploaded and shown.